easybuild/easyconfigs/p/PyBERT/PyBERT-1.7.9-foss-2016a-Python-3.5.1.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'PythonPackage'
+
+name = 'PyBERT'
+version = '1.7.9'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/capn-freako/PyBERT/wiki'
+description = """PyBERT is a serial communication link bit error rate tester (BERT)
+ simulator with a graphical user interface (GUI), written in Python and
+ making use of the Enthought Traits/UI packages. It is intended to give
+ students, hobbyists, and curious engineers the ability to play with
+ serial communication link design concepts. It is not intended as a
+ mission critical tool for use by professional serial communication
+ link designers. There are much better tools available for that purpose."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+patches = [
+    '%(name)s-%(version)s_sphinx.patch',
+    '%(name)s-%(version)s_2to3.patch',
+]
+
+dependencies = [
+    ('Python', '3.5.1'),
+    ('matplotlib', '1.5.1', versionsuffix),
+    ('Pygments', '2.1.3', versionsuffix),
+    ('traits', '4.5.0', versionsuffix),
+    ('traitsui', '5.1.0', versionsuffix),
+    ('enable', '20160817', versionsuffix),
+    ('chaco', '20160817', versionsuffix),
```

Comment:
@hajgato would it make sense to make this a `Bundle` of Python packages (cfr. `matplotlib`), to avoid a bunch of modules/easyconfigs?